rooms.js

```diff
@@ -1478,7 +1478,7 @@ class ChatRoom extends Room {
 		let buffer = '';
 		let counter = 0;
 		for (let i in this.users) {
-			if (!this.users[i].named) {
+			if (!Users(this.users[i]) || !this.users[i].named) {
```

Comment:
`this.users[i]` is a user object...

`.named` tests whether or not the user has a name.